package.json

```diff
@@ -20,8 +20,5 @@
     "nightwatch": "0.9.12",
     "selenium-download": "^2.0.10",
     "superagent": "3.5.2"
-  },
-  "devDependencies": {
-    "browserstack-local": "1.3.0"
   }
```

Comment:
Why did you remove this? This is what connects the local tests to browserstack right? Sorry, don't remember.